src/app/app-routing.module.ts

```diff
@@ -21,12 +22,26 @@ import { WorkspaceCalendarComponent } from './presentation/components/workspace-
 import { TwoSetpVerificationComponent } from './presentation/components/user-components/two-step-verification/two-step-verification.component';
 
 const userWorkspaceRouters: Routes = [
-  { path: 'managment', component: WorkspaceMemberManagmentComponent, pathMatch: 'full'},
+  {
+    path: 'managment',
+    component: WorkspaceMemberManagmentComponent,
+    canActivate: [PermissionGuard],
+    data:{
+      userRoles: [1,2]
```

Comment:
Create tech dept for adding permissions on server side and then integrate it here. Use hard code is a bad solution.